src/js/views/patients/shared/components/owner_component.js

```diff
@@ -116,31 +117,30 @@ export default Droplist.extend({
       currentWorkspaceCache = currentWorkspace.id;
     }
 
-    const currentUser = Radio.request('bootstrap', 'currentUser');
-    const clinicians = getClinicians(currentWorkspace, currentUser);
+    this.currentUser = Radio.request('bootstrap', 'currentUser');
+    this.clinicians = getClinicians(currentWorkspace, this.currentUser);
 
-    if (this.getOption('hasCurrentClinician') && clinicians.get(currentUser)) {
+    if (this.getOption('hasClinicians') && this.clinicians.length) {
       this.lists.push({
-        collection: new Backbone.Collection([currentUser]),
-      });
-    }
-
-    if (this.getOption('hasClinicians') && clinicians.length) {
-      this.lists.push({
-        collection: clinicians,
+        collection: this.clinicians,
         headingText: currentWorkspace.get('name'),
       });
     }
 
     if (this.getOption('hasTeams')) {
       this.lists.push({
-        collection: getTeams(currentWorkspace, currentUser),
+        collection: getTeams(currentWorkspace, this.currentUser),
         headingText: this.lists.length ? i18n.teamsHeadingText : null,
       });
     }
 
     this.setState({ selected: owner });
   },
+  onPicklistSelect({ model }) {
+    this.setState('selected', model || this.currentUser);
+
+    this.popRegion.empty();
+  },
```

Comment:
I couldn't get this to work without overriding this `Droplist` function.

If I don't override this, it's called before the `onChangeSelected()` function below. Which makes things go 💥 since this owner component then is attempting to access a `selected` object that is `null`.

Reply:
So when I attempted this: 

```js
onChangeSelected(selected) {
  if (!selected) {
    this.setState({ selected: currentUser });
      return;
  }

  this.triggerMethod('change:owner', selected);
},
```

The app would 💥 before this function even got the chance to run.

Reply:
I think that's probably because of `const isTeam = selected.type === 'teams';` but this seems ok to me.